src/protocolgame.cpp

```diff
@@ -2762,7 +2763,7 @@ void ProtocolGame::sendAddCreature(const Creature* creature, const Position& pos
 		// screen
 		if (stackpos >= MAX_STACKPOS) {
 			// @todo: should we avoid this check?
-			if (const Tile* tile = creature->getTile()) {
+			if (auto tile = creature->getTile()) {
```

Comment:
Could this be `const auto`
```suggestion
			if (const auto tile = creature->getTile()) {
```